databricks/koalas/frame.py

```diff
@@ -2503,12 +2509,12 @@ def transform(self, func, axis=0, *args, **kwargs):
         1  1  4
         2  4  9
 
-        >>> df.transform(lambda x: x ** 2)  # doctest: +NORMALIZE_WHITESPACE
+        >>> df.transform(abs)  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
How about `(df * -1).transform(abs)` to clearly show built-in function `abs` actually works ??